src/pages/Proposal.vue

```diff
@@ -245,9 +245,9 @@ export default defineComponent({
 						}
 					}
 
-					// check if asset is already opted
+					// check if app is already opted
 					const isApplicationAlreadyOpted = await isApplicationOpted(
-						this.walletStore.address,
+						lsig.address(),
```

Comment:
Why are we using lsig here, shouldn't it be specific to user connected, to check if he/she has opted the app

Reply:
I have checked the add propsal script. It checks if it has been opted by lsig or not. So lsig will be here not the wallet address.

`// check if app is already opted
					const isApplicationAlreadyOpted = await isApplicationOpted(
						lsig.address(),
						this.daoStore.dao_id as number
					);
					// optin
					if (!isApplicationAlreadyOpted) {
						await this.optInLsigToApp(lsig);
					}`

check here we are trying to opt in lsig

Reply:
okay